frontend/src/components/BlogsList.tsx

```diff
@@ -33,8 +30,8 @@ const BlogsList = () => {
   useEffect(() => {
     if (!loading && blogs.length > 0) {
       const timer = setTimeout(() => {
-      setShowEndMessage(true);
-      setShowConfetti(true);
+        setShowEndMessage(true);
+        setShowConfetti(true);
```

Comment:
Please fix formatting issue here

Reply:
![Screenshot 2024-06-17 132016](https://github.com/aadeshkulkarni/figuringout/assets/107796595/791c1c52-2236-45b1-9312-8f5bf09935c7)
Ok so if now i am running npm run lint-fix, i am facing these errors. do you know anything about these errors?

Reply:
npm run lint-fix tried this also, the first command i ran in the screenshot is this one only.

Reply:
Thank you that solved the issue which was happening in AnimatedMessage.tsx. but still I am getting an error which is related to Contributor.tsx. Do you have any solution for this?
![Screenshot 2024-06-17 133148](https://github.com/aadeshkulkarni/figuringout/assets/107796595/5d6d93a8-b071-4d57-814b-ae94f3a07771)

Reply:
Contributor.tsx Ln:73

```javascript
 <ContributorCard key={contributor?.id} contributor={contributor} />
```

Reply:
Thank you for helping out. Also the formatting issues mentioned in this thread are maybe supposed to be that way because i also ran the linting and formatting script but it is still the same.
![Screenshot 2024-06-17 133944](https://github.com/aadeshkulkarni/figuringout/assets/107796595/6f8ba56d-8ef1-4194-b14e-5076a1f00ab0)